### PR TITLE
Change Error message namespace + policy names

### PR DESCRIPTION
**Pull request by yiraeChristineKim**
Description: Change error message "the combined length of the policy namespace and name . cannot exceed 63 characters" to the combined length of the policy namespace and name . cannot exceed 62 characters
Ref: https://issues.redhat.com/browse/ACM-6838

### CODE REVIEW

File: frontend/public/locales/en/translation.json [hunk 1]
```diff
@@ -2638,7 +2638,7 @@
   "The cluster name is already used by another cluster.": "The cluster name is already used by another cluster.",
   "The cluster pool deletion will be blocked until all claimed cluster(s) are deleted.": "The cluster pool deletion will be blocked until all claimed cluster(s) are deleted.",
   "The cluster pool will not generate machine pools if the count is not equal to 3. Node scaling will be unavailable.": "The cluster pool will not generate machine pools if the count is not equal to 3. Node scaling will be unavailable.",
-  "The combined length of namespace and policy name (namespaceName.policyName) must not exceed 63 characters": "The combined length of namespace and policy name (namespaceName.policyName) must not exceed 63 characters",
+  "The combined length of namespace and policy name <namespaceName>.<policyName> must not exceed 62 characters": "The combined length of namespace and policy name <namespaceName>.<policyName> must not exceed 62 characters",
```
Comment (dhaiducek) [inline frontend/public/locales/en/translation.json:2641]:
```suggestion
  "The combined length of the policy namespace and name must not exceed 62 characters": "The combined length of policy namespace and name must not exceed 62 characters",
```

**Review by dhaiducek — DISMISSED** [review]

File: frontend/public/locales/ko/translation.json [hunk 2]
```diff
@@ -2628,7 +2628,7 @@
   "The cluster has no nodes.": "클러스터에 노드가 없습니다.",
   "The cluster name is already used by another cluster.": "클러스터 이름이 이미 다른 클러스터에서 사용되고 있습니다.",
   "The cluster pool will not generate machine pools if the count is not equal to 3. Node scaling will be unavailable.": "클러스터 풀은 개수가 3개가 아닌 경우 머신 풀을 생성하지 않습니다. 노드 확장을 사용할 수 없게 됩니다.",
-  "The combined length of namespace and policy name (namespaceName.policyName) must not exceed 63 characters": "네임스페이스 및 정책 이름(namespaceName.policyName)의 결합된 길이는 63자를 초과해서는 안 됩니다",
+  "The combined length of the policy namespace and name must not exceed 62 characters": "네임스페이스 및 정책 이름이 결합된 길이는 62자를 초과해서는 안 됩니다",
```
Comment (yiraeChristineKim) [inline frontend/public/locales/ko/translation.json:2631]: **'이(가)** 결함된' is more natural Korean

**Review by mprahl — APPROVED** [review]

**Comment by mprahl** [comment]
@yiraeChristineKim could you please run prettier to fix the CI failure?

**Review by dhaiducek — APPROVED** [review]

**Review by chenz4027 — APPROVED** [review]
/lgtm

**Pull request merged**